os/include/tinyara/sched.h

```diff
@@ -370,6 +376,9 @@ struct task_group_s {
 #if !defined(CONFIG_DISABLE_PTHREAD) && defined(CONFIG_SCHED_HAVE_PARENT)
 	pid_t tg_task;				/* The ID of the task within the group      */
 #endif
+#ifdef CONFIG_BINFMT_LOADABLE
+	int tg_loadflag;			/* See LOAD_FLAG_* definitions             */
```

Comment:
If you want to use this value to flag, you'd better use uint32_t instead of int.